src/gql/mutations/save-repo-settings-for-section.graphql

```diff
@@ -0,0 +1,10 @@
+mutation SaveRepoSettingsForSection(
+  $repoSettings: RepoSettingsInput!
+  $section: ProjectSettingsSection!
+) {
+  saveRepoSettingsForSection(repoSettings: $repoSettings, section: $section) {
+    projectRef {
+      id
```

Comment:
We can just import the same fragment  that the `get-repo-settings` query uses. To fetch the same fields in the mutation response.